lib/alma/fine_set.rb

```diff
@@ -17,7 +17,8 @@ def key
     end
 
     def list
-      fetch(key, [])
+      @response.fetch(key, []).map{|item| Alma::AlmaRecord.new(item)}
```

Comment:
Since we are using the Enumerable mixin, should't we be doing this bit inside of the each mehod?